app/javascript/components/explore/ExploreDisplayTabs.js

```diff
@@ -442,34 +442,32 @@ export default function ExploreDisplayTabs(
 
 /** return an array of the tabs that should be shown, given the exploreParams and exploreInfo */
 export function getEnabledTabs(exploreInfo, exploreParams) {
-  let isGeneList =  !!exploreParams.geneList
-  let isMultiGene = exploreParams?.genes?.length > 1
-  let isGene = exploreParams?.genes?.length > 0
-  let isConsensus = !!exploreParams.consensus
-  let hasSpatialGroups = exploreInfo && exploreInfo?.spatialGroups?.length > 0
-  let hasGenomeFiles = exploreInfo && exploreInfo?.bamBundleList?.length > 0
+  const isGeneList = !!exploreParams.geneList
+  const isMultiGene = exploreParams?.genes?.length > 1
+  const isGene = exploreParams?.genes?.length > 0
+  const isConsensus = !!exploreParams.consensus
+  const hasSpatialGroups = exploreInfo && exploreInfo?.spatialGroups?.length > 0
+  const hasGenomeFiles = exploreInfo && exploreInfo?.bamBundleList?.length > 0
   let enabledTabs = []
   if (isGeneList) {
     enabledTabs = ['heatmap']
-  } else {
-    if (isGene) {
-      if (isMultiGene) {
-        if (isConsensus) {
-          enabledTabs = ['scatter', 'distribution', 'dotplot']
-        } else if (hasSpatialGroups) {
-          enabledTabs = ['spatial', 'dotplot', 'heatmap']
-        } else {
-          enabledTabs = ['dotplot', 'heatmap']
-        }
+  } else if (isGene) {
```

Comment:
Probably not in the context of this PR, but I really think we should refactor this method for readability/maintainability.  In context with #922 we now have 8 different tabs that can be rendered at any given time, and the cyclomatic complexity of this is such I had no idea what the logic was going to be here without a serious 10 minutes of inspection and copying snippets out to a text document to assemble what it "would" be after the merge.  This clearly isn't going to scale well, so I think it's time for a refactor.